tools/scale-testing/QidoQueryGenerator/Program.cs

```diff
@@ -34,15 +34,15 @@ public static void Main(string[] args)
                 string namePrefix = pI.Name.Split('^')[0].Substring(0, 3);
                 string startDate = DateTime.Parse(pI.PerformedProcedureStepStartDate).AddMonths(-1).ToString("yyyyMMdd");
                 string endDate = DateTime.Parse(pI.PerformedProcedureStepStartDate).AddMonths(1).ToString("yyyyMMdd");
-                string studyGenericVal = $"/Studies?PatientName={namePrefix}&StudyDate={startDate}-{endDate}&fuzzyMatching=true";
-                string studySpecificVal = $"/Studies?PatientID={pI.PatientId}";
-                string seriesGenericVal = $"/Series?Modality={pI.Modality}";
-                string seriesSpecificVal = $"/Series?Modality={pI.Modality}&PatientID={pI.PatientId}";
+                string studyGenericVal = $"PatientName={namePrefix}&StudyDate={startDate}-{endDate}&fuzzyMatching=true";
+                string studySpecificVal = $"PatientID={pI.PatientId}";
+                //string seriesGenericVal = $"/Series?Modality={pI.Modality}";
```

Comment:
Should these paths be commented out?

Reply:
Yes, because the url was targeting the generic QueryAsync(). So for now the scale test will target Study level query.